src/main/resources/liquibase/042-updateDB.xml

```diff
@@ -12,13 +12,74 @@
-</databaseChangeLog>
\ No newline at end of file
+
+    <changeSet id="carrier_per_study_key" author="pegah">
+        <addColumn tableName="carrier_service">
+            <column name="carrier_tracking_url" type="VARCHAR(500)"></column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="DDP-5445_ups_shipment_table" author="pegah">
+        <createTable tableName="ups_shipment">
+            <column name="ups_shipment_id" type="INT" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"></constraints>
+            </column>
+            <column name="dsm_kit_request_id" type="INT">
+                <constraints nullable="false"></constraints>
+            </column>
+        </createTable>
+
+    </changeSet>
+    <changeSet id="DDP-5445-ups_package-table" author="pegah">
+        <createTable tableName="ups_package">
+            <column name="ups_package_id" type="INT" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"></constraints>
+            </column>
+            <column name="dsm_kit_request_id" type="INT">
+                <constraints nullable="false"></constraints>
+            </column>
+            <column name="ups_shipment_id" type="INT">
+                <constraints nullable="false"></constraints>
+            </column>
+            <column name="tracking_number" type="varchar(50)"></column>
+            <column name="delivery_date" type="varchar(10)"></column><!-- Format: MMDDYYYY-->
+            <column name="delivery_date_type" type="varchar(3)"></column><!-- Indicates the type of delivery-->
+            <column name="delivery_time_start" type="varchar(6)"></column><!-- Format: HHMMSS 24 hour-->
+            <column name="delivery_time_end" type="varchar(6)"></column><!-- Format: HHMMSS 24 hour-->
+            <column name="delivery_time_type" type="varchar(3)"></column><!-- Indicates the type of delivery-->
+
+        </createTable>
+    </changeSet>
+    <changeSet id="DDP-5445_ups_activity_table" author="pegah">
+        <createTable tableName="ups_activity">
+            <column name="ups_activity_id" type="INT" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"></constraints>
+            </column>
+            <column name="ups_package_id" type="INT">
+                <constraints nullable="false"></constraints>
+            </column>
+            <column name="dsm_kit_request_id" type="INT">
+                <constraints nullable="false"></constraints>
+            </column>
+            <column name="ups_location" type="TEXT(10000)"></column>
+            <column name="ups_status_type" type="VARCHAR(10)"></column>
+            <column name="ups_status_description" type="VARCHAR(100)"></column>
+            <column name="ups_status_code" type="VARCHAR(10)"></column>
+            <column name="ups_activity_date" type="VARCHAR(10)"></column>
```

Comment:
The date and time columns can be combined into a single `datetime` field for full precision and ease of querying.